src/test/librbd/test_mock_fixture.h

```diff
@@ -25,7 +25,11 @@ ACTION_P(CopyInBufferlist, str) {
 }
 
 ACTION_P2(CompleteContext, r, wq) {
+#if defined(__clang__)
+  ContextWQ *context_wq = static_cast<ContextWQ *>(wq);
```

Comment:
@tchaikov 

The result will stay the same, but for userspace includes you'd want to look at:
[https://github.com/freebsd/freebsd/blob/386ddae58459341ec567604707805814a2128a57/include/stddef.h](https://github.com/freebsd/freebsd/blob/386ddae58459341ec567604707805814a2128a57/include/stddef.h).
Because sys/sys are the kernel include files, and they can differ.